app/models/contact.rb

```diff
@@ -0,0 +1,13 @@
+class Contact < AbstractDocument
+  def self.tag_metadata_keys
+    %w(
+      contact_group
+    )
+  end
+
+  def self.metadata_name_mappings
+    {
+      "contact_group" => "Topic",
```

Comment:
I don't either, but this is what it's called in the existing Contacts App index page. I'm open to suggestions though. @jamiecobbett?